website_sale_require_login/views/assets.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- © 2015 Antiun Ingeniería S.L. - Jairo Llopis
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+
+<odoo>
+    
+    <template id="assets_frontend" inherit_id="website_sale.assets_frontend">
```

Comment:
Call the template `demo_assets_frontend` and move the file into `demo` directory.

Reply:
Still missing.